wrappers/winuser.go

```diff
@@ -239,21 +240,38 @@ func GetForegroundWindow() syscall.Handle {
 	return syscall.Handle(r1)
 }
 
-func GetWindowText(hwnd syscall.Handle, buffer *uint16, maxCount int) (int, error) {
+func GetWindowText(hwnd syscall.Handle, buffer *uint16, maxCount int32) (int, error) {
 	r1, _, e1 := syscall.Syscall(
-		procGetWindowText.Addr(),
+		procGetWindowTextW.Addr(),
 		3,
-		uintptr(hwnd), uintptr(unsafe.Pointer(buffer)),
+		uintptr(hwnd),
+		uintptr(unsafe.Pointer(buffer)),
 		uintptr(maxCount))
 	if e1 != ERROR_SUCCESS {
 		return int(r1), e1
 	}
 	return int(r1), nil
 }
 
-func GetWindowThreadProcessId(hwnd syscall.Handle, processID *uint32) syscall.Handle {
-	r1, _, _ := syscall.Syscall(procGetWindowThreadProcessId.Addr(), 2, uintptr(hwnd), uintptr(unsafe.Pointer(processID)), 0)
-	return syscall.Handle(r1)
+func GetWindowTextLength(hwnd syscall.Handle) (int32, error) {
+	r1, _, e1 := syscall.Syscall(
+		procGetWindowTextLengthW.Addr(),
+		1,
+		uintptr(hwnd),
+		0,
+		0)
+	if e1 != ERROR_SUCCESS {
+		return int32(r1), e1
+	}
+	return int32(r1), nil
+}
+
+func GetWindowThreadProcessID(hwnd syscall.Handle, processID *uint32) (uint32, error) {
```

Comment:
In `wrappers`, we should match the name of the API function exactly, so `GetWindowThreadProcessId`.  Microsoft apparently chose to use different conventions on different teams, so the kernel has `GetCurrentProcessId` and `GetCurrentThreadId` while the UI has `GetMenuItemID` and `GetDlgCtrlID`.  It was probably a mistake to do that, but it's one that can't be corrected without breaking compatibility, so the Win32 API is stuck with it.  However, it our Go-friendly library, we can use consistent naming, so we do that there while matching the API itself here.